src/Core/System/SalesChannel/Context/AbstractSalesChannelContextFactory.php

```diff
@@ -11,7 +11,7 @@ abstract class AbstractSalesChannelContextFactory
     abstract public function getDecorated(): AbstractSalesChannelContextFactory;
 
     /**
-     * @param array<string, mixed> $options
+     * @param array<string, string|array<string,bool>|null> $options
```

Comment:
really awesome would be a description of the allowed array keys :grin:

Reply:
Since Commercial extends this by additional array keys (`employeeId`, we cannot make sure, that this list is complete. Therefore, I would not provide it.

Reply:
ah okay, true :+1: